tests/test_board_helper.py

```diff
@@ -0,0 +1,138 @@
+from unittest import TestCase, mock
+from unittest.mock import patch, MagicMock
+from core.ships import Ships
+from helpers.board_helper import BoardHelper
+
+class TestBoardHelper(TestCase):
```

Comment:
These test are good, but seem kind of silly since you are just testing static constants. I'd expect them to make more sense if these functions were more configurable, i.e. you could place a ship where you wanted based on some argument or something.